checkov/terraform/checks/resource/openstack/ComputeInstanceAdminPassword.py

```diff
@@ -0,0 +1,22 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_negative_value_check import BaseResourceNegativeValueCheck
+from checkov.common.models.consts import ANY_VALUE
+
+
+class ComputeInstanceAdminPassword(BaseResourceNegativeValueCheck):
+    def __init__(self):
+        name = "Ensure that instance does not use basic credentials"
+        id = "CKV_OPENSTACK_4"
+        supported_resources = ['openstack_compute_instance_v2']
+        categories = [CheckCategories.GENERAL_SECURITY]
```

Comment:
```suggestion
        categories = [CheckCategories.SECRETS]
```
just thinking, if secrets category fits here better 🤔